### ISSUE TITLE
The reasoning behind NTP maxpoll 17

### ISSUE THREAD

**Issue by konstruktoid** [OpenSCAP/scap-security-guide#1982]
According to xccdf_org.ssgproject.content_rule_ntp_set_maxpoll:
```
The maxpoll should be configured to 17 in /etc/ntp.conf to continuously poll NTP servers. 
```

But https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/s1-Configure_NTP.html#s2_Configuring_the_Poll_Interval states that:
```
The default value for maxpoll is 10, which equates to 1024s. Allowed values are in the range 3 to 17 inclusive, which equates to 8s to 36.4h respectively. These options are for use with the peer or server. Setting a shorter maxpoll may improve clock accuracy.
```

What's the reasoning behind setting `maxpoll 17` when the default `maxpoll 10` may improve clock accuracy?

**Comment by redhatrises** [comment]
The reasoning is that that is the value that DISA wants. However, the OVAL check will pass if it is less than or equal to 17, so you can still set it to 10 and pass.

**Comment by konstruktoid** [comment]
The reason I brought it up is because according to U_Red_Hat_Enterprise_Linux_7_STIG_V1R1_Manual-xccdf.xml:
```
If the option is set to "17" or is not set, this is a finding.
[...]
Fix Text: Edit the "/etc/ntp.conf" file and add or update an entry to define "maxpoll" to "10" as follows:

maxpoll 10
```

**Comment by redhatrises** [comment]
This is an error in DISA content that we need to get them to fix/clarify. Throwing in the DISA Content Issues bucket.

**Comment by shawndwells** [comment]
On 5/3/17 9:21 AM, Thomas Sjögren wrote:
>
> The reason I brought it up is because according to
> U_Red_Hat_Enterprise_Linux_7_STIG_V1R1_Manual-xccdf.xml:
>
> |If the option is set to "17" or is not set, this is a finding. [...]
> Fix Text: Edit the "/etc/ntp.conf" file and add or update an entry to
> define "maxpoll" to "10" as follows: maxpoll 10 |

 @redhatrises is spot on - it's something DISA mandated for their STIG
profile. Not something that was recommended by NIST, DoD, Red Hat, the
DoD RHEL consensus group, or the broader RHEL DoD user community.

There are several mismatched rule titles vs check text items on their
content. When we've asked DISA before, they defer to what is requested
in the rule title (in this case, maxpoll 17.

DISA can/should be appropriately queried on this for their official
guidance. Their contact info:
http://iase.disa.mil/stigs/Pages/contact.aspx

**Comment by shawndwells** [comment]
On 5/3/17 9:38 AM, redhatrises wrote:
> This is an error in DISA content that we need to get them to
> fix/clarify. Throwing in the DISA Content Issues bucket.

appreciate that. planning to sync with them post-summit

**Comment by konstruktoid** [comment]
But shouldn't the DISA STIG OpenScap profile follow the DISA Red Hat Enterprise Linux 7 STIG?
`"OpenSCAP Evaluation Report -Guide to the Secure Configuration of Red Hat Enterprise Linux 7
with profile DISA STIG for Red Hat Enterprise Linux 7"` says that `The maxpoll should be configured to 17 in /etc/ntp.conf` but `"Red Hat Enterprise Linux 7 Security Technical Implementation Guide Version: 1 Release: 1 27 Feb 2017"` states that it should be `maxpoll 10`.

**Comment by shawndwells** [comment]
The DISA content has the following regarding testing for pass/fail:

`````
<check-content>Check to see if NTP is running in continuous mode.

# ps -ef | grep ntp

If NTP is not running, this is a finding.

If the process is found, then check the "ntp.conf" file for the "maxpoll" option setting:

# grep maxpoll /etc/ntp.conf

maxpoll 17

If the option is set to "17" or is not set, this is a finding.
......
</check-content>
`````

The OpenSCAP/SSG OVAL is aligned with DISA's request to fail people if maxpoll != 17. Even though other elements of the DISA content say to use ``maxpoll 10``.

**Comment by konstruktoid** [comment]
Thanks for clarifying @shawndwells, feel free to close unless we want to wait for the DISA response.

**Comment by redhatrises** [comment]
Resolved in #2252
